web/src/regression/core.test.ts

```diff
@@ -76,7 +76,7 @@ describe('Core functionality regression test suite', () => {
             await driver.page.waitForSelector('.e2e-settings-file .monaco-editor')
             return driver.page.evaluate(() => {
                 const editor = document.querySelector('.e2e-settings-file .monaco-editor') as HTMLElement
```

Comment:
This element should work and not include the `<style>`:
```suggestion
                const editor = document.querySelector<HTMLElement>('.e2e-settings-file .monaco-editor .view-lines')
```

Reply:
cool, thanks felix. will use :-)